reverse_engineering/helpers/snowflakeHelper.js

```diff
@@ -440,7 +467,7 @@ const getDDL = async (tableName, logger) => {
 		return getFirstObjectItem(_.first(queryResult));
 	} catch (err) {
 		logger.log('error', { tableName, message: err.message, stack: err.stack }, 'Getting table DDL')
-		throw err;
+		return ''
```

Comment:
if user have no rights to get ddl - we still able to parse table/view and not have errors, but in logs there will be messages about this error